lib/queries.js

```diff
@@ -154,3 +158,44 @@ export const deleteAnnotated = async (uris) => {
 
   await update(deleteQuery);
 };
+
+/**
+ * Fetches all the template uris linked to a array of given uris
+ *
+ * @param {string[]} uris - The array containing the uris of the templates.
+ * @returns {string[]} An array of all the uris of the linked templates.
+ */
+export const getLinkedTemplates = async (templateUris) => {
+  if (templateUris && !Array.isArray(templateUris)) {
+    throw new Error("templateUris must be an array of strings.");
+  }
+
+  const values = templateUris?.length
+    ? `VALUES ?uri {${templateUris
+        .map((uri) => sparqlEscapeUri(uri))
+        .join(" ")}}`
+    : "";
+
+  const selectTemplateQuery = `
+    PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+    PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX prov: <http://www.w3.org/ns/prov#>
+    PREFIX mobiliteit: <https://data.vlaanderen.be/ns/mobiliteit#>
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+  
+    SELECT DISTINCT ?linkedTemplateUri WHERE {
```

Comment:
afaict when templateUris is empty, this returns _all_ templates linked to any other template, cause there's no constraint on the `?uri` variable in the query. Is this intentional?